src/main/java/org/apache/maven/shared/utils/io/FileUtils.java

```diff
@@ -161,47 +161,52 @@ protected FileUtils()
      * Returns the directory path portion of a file specification string.
      * Matches the equally named unix command.
      *
-     * @param filename the file path
+     * @param path the file path
      * @return the directory portion excluding the ending file separator
+     * @deprecated use {@code Paths.get(path).getParent().getName()}
```

Comment:
Does this advise also work when the path is relative? Haven't tested.

Reply:
It should.